resources/views/fields/image.blade.php

```diff
@@ -1,4 +1,7 @@
-<div class="{{ $field->divClass ?? config('livewire-forms.defaults.divClass') }}">
+<div @class([
+    $field->divClass ?? config('livewire-forms.defaults.divClass'),
+    $field->colClass ?? config('livewire-forms.defaults.colClass')])
+>
```

Comment:
Ik kan niet op de correcte lijn commenten omdat die niet aangepast is, maar: evt. de .img-fluid class ook nog in de config steken?

Reply:
gefixed in [1c6b651](https://github.com/codedor/laravel-livewire-forms/pull/9/commits/1c6b6513a5df1843a96189a5af66113236c4d9cf)